src/lib/text-preview-helper.js

```diff
@@ -0,0 +1,18 @@
+//Create a classname for the text preview position while previewing the position of text on a GIF before download.
+const getTextPosition = (textAlign, textBaseline) => {
```

Comment:
Can be accomplished with:

```return `${textBaseline}-${textAlign}`;```

This will require you to make a minor change of setting `bottom-right` as the default in the form itself, which is fine. Do away with the unselected 'Pick a Position' and just have it naturally be your `bottom-right`.